### PR TITLE
chore: update scripts for validating HugeGraph 1.2.0 release

### PR DESCRIPTION
**Pull request by VGalaxies**
This PR is a subtask of https://github.com/hugegraph/actions/issues/5, and it includes the following changes:

1. Synchronize the content of the `validate-release.sh` and `validate-release.yml` with https://github.com/hugegraph/actions/blob/master/.github/workflows/validate-release.yml.
2. Fix the startup of server from compiled packages.
3. Optimize some processes and comments.

Currently, steps 7 (validate the binary packages) and 8 (run the binary packages in the server) in this script have detected some issues that await further resolution.

### CODE REVIEW

**Comment by VGalaxies** [comment]
@imbajin PTAL~

File: .github/configs/settings.xml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+                    <url>https://repository.apache.org/content/groups/staging/</url>
+                </pluginRepository>
+            </pluginRepositories>
+        </profile>
+    </profiles>
+
+    <activeProfiles>
+        <activeProfile>local-repo</activeProfile>
+    </activeProfiles>
+</settings>
```
Comment (imbajin) [inline .github/configs/settings.xml:60]:
```suggestion
</settings>

```

**Review by imbajin — COMMENTED** [review]
Good job, better to add Category B for **source code** LICENSE check

File: .github/workflows/validate-release.yml [hunk 2]
```diff
@@ -198,7 +199,8 @@ jobs:
           cd ./*hubble*${{ inputs.release_version }} || exit
           # TODO: add hubble doc & test it
           cat conf/hugegraph-hubble.properties
-          bin/start-hubble.sh && bin/stop-hubble.sh
+          bin/start-hubble.sh || exit
+          bin/stop-hubble.sh || exit
```
Comment (imbajin) [inline .github/workflows/validate-release.yml:203]: @liuxiaocs7 @aroundabout could we add some basic curl check like `curl localhost:8088/xxxAPI` to ensure the hubble runs well?

Comment (liuxiaocs7) [reply .github/workflows/validate-release.yml:203]: I remember that `start_hubble.sh` included the check:

https://github.com/apache/incubator-hugegraph-toolchain/blob/773469c069bc79e9e8159349a5b086f6da3f97eb/hugegraph-hubble/hubble-dist/assembly/static/bin/start-hubble.sh#L83-L94

Comment (imbajin) [reply .github/workflows/validate-release.yml:203]: > I remember that `start_hubble.sh` included the check:
> 
> [apache/incubator-hugegraph-toolchain@`773469c`/hugegraph-hubble/hubble-dist/assembly/static/bin/start-hubble.sh#L83-L94](https://github.com/apache/incubator-hugegraph-toolchain/blob/773469c069bc79e9e8159349a5b086f6da3f97eb/hugegraph-hubble/hubble-dist/assembly/static/bin/start-hubble.sh#L83-L94)

seems it only check server but not hubble itself (it's good to enhance the check in `start-hubble` later), currently we could check it in our script

**Review by imbajin — APPROVED** [review]

File: .github/workflows/validate-release.yml (hunk 2 above)
Comment (VGalaxies) [reply .github/workflows/validate-release.yml:210]: @Pengzna here can be improved in next PR~

**Comment by javeme** [comment]
related #197

**Pull request merged**